azure-pipelines.yml

```diff
@@ -50,7 +50,8 @@ stages:
     - script: |
         set -ex
         sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev
-        sudo dpkg -i ../target/debs/buster/{libyang_1.0.73_amd64.deb,libswsscommon_1.0.0_amd64.deb,python3-swsscommon_1.0.0_amd64.deb,libnl-3-200_*.deb,libnl-genl-3-200_*.deb,libnl-nf-3-200_*.deb,libnl-route-3-200_*.deb,libhiredis0.14_*.deb}
+        sudo apt-get -y install libhiredis0.14
```

Comment:
>libhiredis0 [](http://example.com/codeflow?start=32&length=11)

Do you want to install libhiredis-dev?

Reply:
No, the original pipeline only install libhiredis0.14, I want keeps no change.